src/content/docs/usage/Getting_started/introduction.md

```diff
@@ -7,90 +7,92 @@ parentWeight: 10
 
 ## What is nf-core?
 
-nf-core is a community effort to collect a curated set of analysis pipelines built using [Nextflow](https://www.nextflow.io/docs/latest/index.html).
+nf-core is a community effort to collect a curated set of analysis pipelines built with [Nextflow](https://www.nextflow.io/docs/latest/index.html).
 
 nf-core has three target audiences: facilities, single users, and developers.
-For facilities, it provides highly automated and optimised pipelines that guarantee reproducibility of results for their users.
-Single users profit from portable, documented, and easy-to-use workflows.
-However, you can also become a developer and write your own pipelines in Nextflow using ready-made templates and helper tools.
+For facilities, it provides highly automated and optimized pipelines that guarantee reproducibility of results for users.
```

Comment:
I think @ewels would agree with me for keeping 🇬🇧 spelling here, unless there has been a discussion elsewhere?

Reply:
Yes, (unfortunately for my taste) we use British English 🧐